### PR TITLE
Rg user profile

### PR DESCRIPTION
**Pull request by rayguna**
Photogram 4.1 - user profile.

### CODE REVIEW

**Review by kcalva — APPROVED** [review]
Great job. 
I also like the fact that you updated the README to use it for your own needs. Nice

File: app/views/comments/_form.html.erb [hunk 1]
```diff
@@ -11,22 +13,13 @@
     </div>
   <% end %>
 
-  <div>
-    <%= form.label :author_id, style: "display: block" %>
-    <%= form.text_field :author_id %>
-  </div>
+  <%= form.hidden_field :photo_id %>
```
Comment (JoseMDO) [inline app/views/comments/_form.html.erb:16]: Nice job using hidden inputs!

File: app/views/users/show.html.erb [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+          <h5 class="card-title"><%= photo.owner.username %></h5>
+          <p class="card-text"><%= photo.caption %></p>
+        </div>
+        <ul class="list-group list-group-flush">
+        
+          <% photo.comments.each do |comment| %>
+            <li class="list-group-item">
+              <div class="d-flex">
+                <div class="flex-shrink-0">
+                  <%= image_tag photo.image %>
```
Comment (JoseMDO) [inline app/views/users/show.html.erb:22]: Great job using all these helper methods!

File: config/routes.rb [hunk 3]
```diff
@@ -13,4 +13,6 @@
   # Defines the root path route ("/")
   # root "articles#index"
 
+  get "/:username" => "users#show", as: :user
```
Comment (JoseMDO) [inline config/routes.rb:16]: Nice custom route!

**Review by JoseMDO — APPROVED** [review]
Great job Ray, looks good!

**Review by hcontr4 — COMMENTED** [review]
Review aligns with assignment specifications.

**Pull request merged**